app/src/main/java/ch/epfl/sweng/SDP/game/WaitingPageActivity.java

```diff
@@ -134,7 +143,7 @@ public void onCancelled(@NonNull DatabaseError databaseError) {
         }
     };
 
-    private final ValueEventListener listenerWord2 = new ValueEventListener() {
+    protected final ValueEventListener listenerWord2 = new ValueEventListener() {
```

Comment:
Why are the listeners protected now?

Reply:
This way the tests can call the methods from this listener